Makefile.docker

```diff
@@ -0,0 +1,34 @@
+TOPDIR=$(dir $(lastword $(MAKEFILE_LIST)))
+DOCKERFILE_DIR     ?= ./
+DOCKER_REGISTRY    ?= quay.io
+DOCKER_ORG         ?= $(USER)
+DOCKER_TAG         ?= latest
+BUILD_TAG          ?= latest
+RELEASE_VERSION    ?= $(shell cat $(TOPDIR)/release.version)
+
+ifdef DOCKER_ARCHITECTURE
+  DOCKER_PLATFORM = --platform linux/$(DOCKER_ARCHITECTURE)
+  DOCKER_PLATFORM_TAG_SUFFIX = -$(DOCKER_ARCHITECTURE)
+endif
+
+.PHONY: docker_build
+docker_build:
+	# Build Docker image ...
+	docker $(DOCKER_BUILDX) build $(DOCKER_PLATFORM) $(DOCKER_BUILD_ARGS) --build-arg strimzi_mqtt_bridge_version=$(RELEASE_VERSION) -t strimzi/$(PROJECT_NAME):latest $(DOCKERFILE_DIR)
+	# Also tag with $(BUILD_TAG)
+	docker tag strimzi/$(PROJECT_NAME):latest strimzi/$(PROJECT_NAME):$(BUILD_TAG)$(DOCKER_PLATFORM_TAG_SUFFIX)
+
+.PHONY: docker_load
```

Comment:
```suggestion
.PHONY: docker_save
docker_save:
	# Saves the container as TGZ file
	docker save strimzi/$(PROJECT_NAME):$(BUILD_TAG)$(DOCKER_PLATFORM_TAG_SUFFIX) | gzip > mqtt-bridge$(DOCKER_PLATFORM_TAG_SUFFIX).tar.gz

.PHONY: docker_load
```

Added the `docker_save` target which is useful to the image as tar.gz to inspect with tools or load offline (as you have with the `docker_load` target below)